src/main/webapp/app/sharing/sharing.component.html

```
@@ -0,0 +1,121 @@
+<body>
+    <div class="content">
+        <table style="border-spacing: 50px 0px; border-collapse: separate">
+            <thead>
+                <th jhiTranslate="artemisApp.sharing.importedExercise">Imported Exercise</th>
```

HTMLHint (1.5.0)

[error] The attribute name of [ jhiTranslate ] must be in lowercase.

(attr-lowercase)